src/main/java/org/spongepowered/common/mixin/core/server/MixinMinecraftServer.java

```diff
@@ -855,94 +396,51 @@ public boolean unloadWorld(World world) {
 
     @Override
     public Optional<WorldProperties> getDefaultWorld() {
-        if (DimensionManager.getWorldFromDimId(0) != null) {
-            return Optional.of(((World) DimensionManager.getWorldFromDimId(0)).getProperties());
+        final Optional<WorldServer> optWorldServer = DimensionManager.getWorldByDimensionId(0);
```

Comment:
It would be quite neat to use `DimensionManager.getWorldById(0).transform` here, which automatically handles the "if present" check